backend/meson.build

```diff
@@ -32,51 +31,63 @@ backend_files = files(
 	'wayland/tablet_v2.c',
 )
 
-backend_deps = [
-	drm,
-	egl,
-	gbm,
-	libinput,
-	pixman,
-	xkbcommon,
-	wayland_server,
-	wlr_protos,
-	wlr_render,
+# Client protocols used by wayland backend
```

Comment:
Maybe we can move this to `backend/wayland/meson.build`?

Reply:
Actually, I think it would be nicer to have exactly one Meson file per backend.

Reply:
Done